draft-ietf-quic-transport.md

```diff
@@ -2176,9 +2176,10 @@ path is no longer needed (such as the case in {{off-path-forward}}).
 A sender can make exceptions for probe packets so that their loss detection is
 independent and does not unduly cause the congestion controller to reduce its
 sending rate.  An endpoint might set a separate timer when a PATH_CHALLENGE is
-sent, which is cancelled when the corresponding PATH_RESPONSE is received.  If
-the timer fires before the PATH_RESPONSE is received, the endpoint might send a
-new PATH_CHALLENGE, and restart the timer for a longer period of time.
+sent, which is cancelled if the corresponding PATH_RESPONSE is received or a
```

Comment:
It's not clear what the "or" applies to.  You can undo the changes and simply add your new sentence at the end; see my suggestions below.

Reply:
Agreed, that was awkward, thanks for the suggestions.  Accepted.